### PR TITLE
NETOBSERV-1474: change dup mode to use list as the default

### PR DESCRIPTION
**Pull request by msherif1234**
## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: controllers/flp/flp_test.go [hunk 1]
```diff
@@ -712,7 +712,6 @@ func TestConfigMapShouldDeserializeAsJSONWithLokiStack(t *testing.T) {
 		"DstK8S_Type",
 		"K8S_FlowLayer",
 		"_RecordType",
-		"FlowDirection",
```
Comment (jpinsonneau) [inline controllers/flp/flp_test.go:715]: This need to be restored since we keep FlowDirection anyways now

**Review by jpinsonneau — CHANGES_REQUESTED** [review]

File: controllers/flp/flp_test.go (hunk 1 above)
Comment (msherif1234) [reply controllers/flp/flp_test.go]: restored

**Review by jpinsonneau — APPROVED** [review]
LGTM, thanks @msherif1234

**Pull request merged**